spec/vcr_cassettes/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.yml

```diff
@@ -0,0 +1,113 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://iam.cloud.ibm.com/identity/token#apikey
+    body:
+      encoding: UTF-8
+      string: apikey=IBM_CLOUD_POWERVS_API_KEY&grant_type=urn%3Aibm%3Aparams%3Aoauth%3Agrant-type%3Aapikey
+    headers:
+      User-Agent:
+      - OpenAPI-Generator/1.0.2/ruby
+      Content-Type:
+      - application/x-www-form-urlencoded
+      Accept:
+      - application/json
+      Expect:
+      - ''
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      X-Content-Type-Options:
+      - nosniff
+      Ibm-Cloud-Service-Name:
+      - iam-identity
+      X-Correlation-Id:
+      - bmxyMjg-5a6dbf6c6db64c8296bfe11c3c5ddd2f
+      Cache-Control:
+      - no-cache, no-store, must-revalidate
+      Expires:
+      - '0'
+      Pragma:
+      - no-cache
+      Content-Type:
+      - application/json
+      Content-Language:
+      - en-US
+      Content-Length:
+      - '1635'
+      Strict-Transport-Security:
+      - max-age=31536000; includeSubDomains
+      Connection:
+      - keep-alive
+      Akamai-Grn:
+      - 0.513b2f17.1750866320.3934c864
+      X-Proxy-Upstream-Service-Time:
+      - '202'
+    body:
+      encoding: ASCII-8BIT
+      string: '{"access_token":"ACCESS_TOKEN","refresh_token":"REFRESH_TOKEN","ims_user_id":"22222","token_type":"Bearer","expires_in":3600,"expiration":4102466400,"scope":"ibm
+        openid"}'
+  recorded_at: Wed, 25 Jun 2025 15:45:21 GMT
+- request:
+    method: get
+    uri: https://resource-controller.cloud.ibm.com/v2/resource_instances/IBM_CLOUD_POWERVS_INSTANCE_ID
+    body:
+      encoding: ASCII-8BIT
+      string: ''
+    headers:
+      User-Agent:
+      - ibm_cloud_resource_controller-ruby-sdk-2.0.3 x86_64-pc-linux ruby-3.3.8
+      X-Ibmcloud-Sdk-Analytics:
+      - service_name=resource_controller;service_version=V2;operation_id=get_resource_instance
+      Accept:
+      - application/json
+      Authorization: Bearer xxxxxx
+      Connection:
+      - close
+  response:
+    status:
+      code: 34
```

Comment:
I have been unable to get a `34` response code when making this API call via `curl` and the same headers, etc.